tee-aps/sealed-bid-auction/src/api/controller/AuctionController.ts

```diff
@@ -0,0 +1,8 @@
+import type {Request, Response} from "express";
```

Comment:
When I mentioned using Bun for it I meant using the built in http server for performance.
Not a big deal though, express is not the fastest but for poc it will be okay.
But I have a feeling that we will keep it for longer than this, if we want to secure headroom for performance (in case we won't have time for rewrite in rust) the time is now to switch as it's super trivial still.